app/src/utils/Utils.test.ts

```diff
@@ -412,3 +413,30 @@ describe('downloadFile', () => {
     expect(anchor.remove).toHaveBeenCalled();
   });
 });
+
+describe('pluralize', () => {
+  it('pluralizes a word', () => {
+    const response = pluralize(2, 'apple');
+    expect(response).toEqual('apples');
+  });
+
+  it('pluralizes a word with undefined quantity', () => {
+    const response = pluralize(null as unknown as number, 'orange');
+    expect(response).toEqual('oranges');
+  });
```

Comment:
This is hardly critical, and entirely subjective, so feel free to ignore: my instinct with a null/undefined value would be that it just returns the original word unchanged (when in doubt, do nothing).

Reply:
In my opinion, the word should always be plural even if the quantity is undefined, since without this function, the word would typically be written in plural anyway. And since `null` or `undefined` quantities are synonymous with having zero of something, to me it makes sense to pluralize the word the same way you would for a quantity of `0`.